packages/grafana-data/src/utils/location.ts

```diff
@@ -14,10 +14,16 @@ let getVariablesUrlParams: (scopedVars?: ScopedVars) => UrlQueryMap;
 const stripBaseFromUrl = (url: string): string => {
   const appSubUrl = grafanaConfig.appSubUrl ?? '';
   const stripExtraChars = appSubUrl.endsWith('/') ? 1 : 0;
-  const urlWithoutBase =
-    url.length > 0 && url.indexOf(appSubUrl) === 0 ? url.slice(appSubUrl.length - stripExtraChars) : url;
+  const isAbsoluteUrl = !url.startsWith('/');
```

Comment:
If it starts with http its also absolute

Reply:
it covers it, doesn't it?

Reply:
but should it not be: 

const isAbsolute = url.startsWith(`http`)   

I am not sure I understand the function, an url like `datasources/new`  does not starts with `/` , so isAbsolute would be true